src/components/TaskItem/Drawer/Drawer.tsx

```diff
@@ -91,6 +105,7 @@ export function TaskItemDrawer({
           <ListItemButton
             selected={task.status === 'complete'}
             onClick={onClick('complete')}
+            {...getPostLink(task.postComplete)}
```

Comment:
these redirects should get moved as a history.go call in the onClick method as you only want to route to the post form after success of the mutation call to fluro.

Reply:
Refactored to use a dialog so mutation would only trigger on dialog submit